src/arrayReverse.test.js

```diff
@@ -7,14 +7,29 @@ describe(`Function 'arrayReverse':`, () => {
     expect(arrayReverse).toBeInstanceOf(Function);
   });
 
-  it(`should return an array`, () => {
+  it(`should return an array of reversed strings in reversed order`, () => {
+    expect(arrayReverse(['Mate', 'Academy'])).toEqual(['ymed', 'acAetaM']);
+  });
 
+  it(`should return an empty array
+    if original array is empty`, () => {
```

Comment:
The test description is misleading. It should mention an empty array instead of an empty string.